app/views/layouts/exercise_inputs/editors/_multiple_files.html.erb

```diff
@@ -4,15 +4,19 @@
     <span class="files-tabs">
       <ul class="nav nav-tabs">
         <% @files.each_with_index do |file, index| %>
-            <li role="presentation" class="<%= 'active' if index == 0 %>" data-target="#editor-file-<%= index %>" tabindex='0' data-toggle='tab'>
-              <a href="#"><%= file.name %></a>
+            <li role="presentation" class="file-tab <%= 'active' if index == 0 %>" data-target="#editor-file-<%= index %>" tabindex='0' data-toggle='tab'>
+              <a class="file-name" href="#"><%= file.name %></a> <i class="delete-file-button fa fa-times"></i>
             </li>
         <% end %>
       </ul>
+      <i class="add-file-button fa fa-plus"></i>
     </span>
+
+    <input id="highlight-modes" type="hidden" value="<%=@highlight_modes.to_json%>" />
+
     <div class="tab-content">
       <% @files.each_with_index do |file, index| %>
-          <div role="tabpanel" class="tab-pane mu-input-panel <%= 'fade in active' if index == 0 %>" id="editor-file-<%= index %>">
+          <div role="tabpanel" class="file-editor tab-pane mu-input-panel <%= 'fade in active' if index == 0 %>" id="editor-file-<%= index %>">
```

Comment:
@fedescarpa should this be called `mu-file-editor`? @rodri042 does it make sense to the runners to customize it?

Reply:
Hmm. I can't find any good use cases, and also it wouldn't fit the current design, where the 'editor' field can be either 'custom' or  'multiple_files'.